bijection-avro/src/main/scala/com/twitter/bijection/avro/AvroCodecs.scala

```diff
@@ -103,30 +103,53 @@ object SpecificAvroCodecs {
 
   /**
     * Returns Injection capable of serializing and deserializing a compiled avro record using org.apache.avro.io.BinaryEncoder.
+    * @tparam T compiled Avro record
+    * @param schema The record's schema
+    * @return Injection
+    */
+  def toBinary[T <: SpecificRecordBase: ClassTag](schema: Schema): Injection[T, Array[Byte]] = {
+    val writer = new SpecificDatumWriter[T](schema)
+    val reader = new SpecificDatumReader[T](schema)
+    new BinaryAvroCodec[T](writer, reader)
+  }
+
+  /**
+    * Returns Injection capable of serializing and deserializing a compiled avro record using org.apache.avro.io.BinaryEncoder.
     * Fetches the schema from the specified class in order to be compatible with generated Scala classes.
     * @tparam T compiled Avro record
     * @return Injection
     */
   def toBinary[T <: SpecificRecordBase: ClassTag]: Injection[T, Array[Byte]] = {
     val record = classTag[T].runtimeClass.newInstance().asInstanceOf[T]
     val schema = record.getSchema
-    val writer = new SpecificDatumWriter[T](schema)
-    val reader = new SpecificDatumReader[T](schema)
-    new BinaryAvroCodec[T](writer, reader)
+    toBinary(schema)
   }
 
   /**
     * Returns Injection capable of serializing and deserializing a generic avro record using org.apache.avro.io.JsonEncoder to a
     * UTF-8 String
     * @tparam T compiled Avro record
+    * @param schema The record's schema
     * @return Injection
     */
   def toJson[T <: SpecificRecordBase: ClassTag](schema: Schema): Injection[T, String] = {
```

Comment:
I'd live to remove the `ClassTag` here if possible, but that would be a binary incompatible break, so we can leave it as a TODO for now.